### PR TITLE
OKTA-78078: apps api can now create inactive apps

### PR DESCRIPTION
**Pull request by mauriciocastillosilva-okta**
Added the query parameters 'activate', which is a boolean that allows creating apps with INACTIVE status

Reviewers: @federations-okta and @karlmcguinness-okta 
Related to: https://oktainc.atlassian.net/browse/OKTA-78078

Note: first time changing api docs, so let me know if I am missing something

### CODE REVIEW

File: _source/docs/api/resources/apps.md [hunk 1]
```diff
@@ -700,9 +700,10 @@ Adds a new application to your Okta organization.
 ##### Request Parameters
 {:.api .api-request .api-request-params}
 
-Parameter | Description                                | Param Type | DataType                          | Required | Default
---------- | ------------------------------------------ | ---------- | --------------------------------- | -------- | -------
-app       | App-specific name, signOnMode and settings | Body       | [Application](#application-model) | TRUE     |
+Parameter | Description                                                                            | Param Type | DataType                          | Required | Default
+--------- | -------------------------------------------------------------------------------------- | ---------- | --------------------------------- | -------- | -------
+activate  | Executes [activation lifecycle](#activate-application) operation when creating the app | Query      | Boolean                           | FALSE    | TRUE
```
Comment (tthompson-okta) [inline _source/docs/api/resources/apps.md]: Is `activate` a query param or a new part of the Application model?

Comment (mauriciocastillosilva-okta) [inline _source/docs/api/resources/apps.md]: a query param

Comment (tthompson-okta) [inline _source/docs/api/resources/apps.md]: Ah sorry, didn't realize there was a column that say explicitly this.

**Comment by tthompson-okta** [comment]
Looks good! :ship:

**Comment by tthompson-okta** [comment]
@joelfranusic-okta @slee-okta @hassenkaraa-okta @eklein-okta FYI, up to you guys to merge this in. The code has already shipped.

**Pull request merged**